fairlearn/reductions/exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -11,14 +11,12 @@
 
 from __future__ import print_function
 
-__all__ = ["expgrad"]
-
 import numpy as np
 import scipy.optimize as opt
 import pandas as pd
 import pickle
 import functools
-import fairlearn.reductions.moments as moments
+from fairlearn.reductions import moments as moments
```

Comment:
`as moments` redundant here?

Reply:
Correct! This must be some remnant from the original code.

Reply:
I already have a follow-up PR and will address this there. I hope that's okay for you.